src/lib/charts/ChartLegend/chart-legend.css

```diff
@@ -0,0 +1,42 @@
+.lc-chart-legend {
```

Comment:
We shouldn't need this file anymore. I loved pie chart legend styles to the `PieChart/pie-chart-legend.css` file.

If you have made any changes to this file since I merged my updates, then move them over there.

Reply:
The base `lc-chart-legend` styles are in the base `chart.css`